cypress/integration/preserve-auth0-cookies-once.spec.js

```diff
@@ -1,30 +1,50 @@
-/**
- * Cypress automatically clears all cookies before each test to prevent state
- * from building up, that's why we test preserving cookies here.
- */
-context('Cookie', () => {
-  before(() => {
-    cy.login();
-  });
+const tests = [
+  {
+    title: 'Cookie',
+    credentials: undefined,
+  },
+  {
+    title: 'Split Cookie',
+    credentials: {
+      username: Cypress.env('auth0SplitCookieUsername'),
+      password: Cypress.env('auth0SplitCookiePassword'),
```

Comment:
Where do these `Cypress.env` values come from?

Reply:
This credential should be an existing auth0 user whose token generates a large cookie. You can fill up the test user's `app_metadata` with dummy data like permissions array with at least 30 elements and append to the token with an auth0 rule. I tested it with my auth0 configuration so you might need to add even more.

The variables are provided via the `.env` file or GitHub secrets which I forgot to configure at the time u were reviewing 😅

Alternatively, the default or alt user can be configured as well to prevent adding yet another user.